app/views/claims/sampling/claims/error.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-width-container">
```

Comment:
I believe you are missing the page title.
`<% content_for :page_title, t(".page_title") %>`